pages/vote/elo.tsx

```diff
@@ -90,50 +112,67 @@ export default function Elo({ sessions }: EloProps): JSX.Element {
         sessionA={sessionPair.SubmissionA}
         sessionB={sessionPair.SubmissionB}
         onSessionChoice={sessionChoiceHandler}
+        layout={layoutVariant}
       />
 
-      <StyledEloButtonContainer>
-        <StyledVoteButton
-          kind="primary"
-          position="left"
-          onClick={(e) => {
-            e.currentTarget.blur()
-            window.scrollTo(0, 0)
-            sessionChoiceHandler(sessionPair.SubmissionA, sessionPair.SubmissionB, false)
-          }}
-        >
-          Option 1
-        </StyledVoteButton>
-        <StyledVoteButton
-          kind="tertiary"
-          position="centre"
-          onClick={(e) => {
-            e.currentTarget.blur()
-            window.scrollTo(0, 0)
-            sessionChoiceHandler(sessionPair.SubmissionA, sessionPair.SubmissionB, true)
-          }}
-        >
-          It's a Draw!
-        </StyledVoteButton>
-        <StyledVoteButton
-          kind="secondary"
-          position="right"
-          onClick={(e) => {
-            e.currentTarget.blur()
-            window.scrollTo(0, 0)
-            sessionChoiceHandler(sessionPair.SubmissionB, sessionPair.SubmissionA, false)
-          }}
-        >
-          Option 2
-        </StyledVoteButton>
-      </StyledEloButtonContainer>
+      <StyledEloVoteFooter>
+        <StyledEloButtonContainer>
+          <StyledVoteButton
+            kind="primary"
+            position="left"
+            onClick={(e) => {
+              e.currentTarget.blur()
+              window.scrollTo(0, 0)
+              sessionChoiceHandler(sessionPair.SubmissionA, sessionPair.SubmissionB, false)
+            }}
+          >
+            Option 1
+          </StyledVoteButton>
+          <StyledVoteButton
+            kind="tertiary"
+            position="centre"
+            onClick={(e) => {
+              e.currentTarget.blur()
+              window.scrollTo(0, 0)
+              sessionChoiceHandler(sessionPair.SubmissionA, sessionPair.SubmissionB, true)
+            }}
+          >
+            It's a Draw!
+          </StyledVoteButton>
+          <StyledVoteButton
+            kind="secondary"
+            position="right"
+            onClick={(e) => {
+              e.currentTarget.blur()
+              window.scrollTo(0, 0)
+              sessionChoiceHandler(sessionPair.SubmissionB, sessionPair.SubmissionA, false)
+            }}
+          >
+            Option 2
+          </StyledVoteButton>
+        </StyledEloButtonContainer>
+        <StyledLayoutLabel>
+          <input
+            type="checkbox"
+            onChange={() => {
+              setLayoutVariant((value) => (value === 'expanded' ? 'stacked' : 'expanded'))
+            }}
+          />
+          Change layout? <span>{layoutVariant === 'stacked' ? 'Expand abstracts' : 'Stack talks'}</span>
```

Comment:
I don't think you need to change anything, but just wanted to point out that you can't tab to this button. For the reasons below, I don't think it's a big deal, just figured I'd mention in case you have a different opinion.

We're only displaying it on small devices, so it's intended for mobile, which means they're unlikely to tab anyway.

In terms of people using screen readers, I have no idea how this behaves, but if they're using a screen reader I doubt they care too much about the layout.

Reply:
Forgot we had an screen reader only class so it was an easy fix.